### ISSUE TITLE
psutil version issue

### ISSUE THREAD

**Issue by scg34** [uw-ipd/RoseTTAFold2#6]
I had an issue when performing the first (example) prediction after installing as follows:

`ImportError: version conflict: '/usr/local/xtal/miniconda/envs/RF2/lib/python3.10/site-packages/psutil/_psutil_linux.cpython-310-x86_64-linux-gnu.so' C extension module was built for another version of psutil (5.9.0 instead of 5.9.5); you may try to 'pip uninstall psutil', manually remove /usr/local/xtal/miniconda/envs/RF2/lib/python3.10/site-packages/psutil/_psutil_linux.cpython-310-x86_64-linux-gnu.so or clean the virtual env somehow, then reinstall`

Basically an incorrect psutil version installed by pip. Easy to fix, just removed the offending package and then reinstalled using the appropriate version:

```
conda activate RF2
pip uninstall psutil
pip install --ignore-installed --no-cache-dir psutil==5.9.0
```

Posting in case anyone else gets tripped up with the same issue. Might be worth adding:
`psutil==5.9.0`
to SE3Transformer/requirements.txt?

Cheers,

Stephen
